addons/website/models/website.py

```diff
@@ -353,7 +353,7 @@ def set_images(images):
         website.configurator_done = True
 
         # Enable tour
-        tour_asset_id = self.env['ir.asset']._get_related_assets([('name', '=', 'website.configurator_tour')])
+        tour_asset_id = self.env['ir.asset']._get_related_assets([('key', '=', 'theme_common.configurator_tour'), ('website_id', '=', website.id)])
```

Comment:
theme_common is not a dependence of website.

Reply:
Mmmh so I imagine that I should move bakc all the stuffs I put in theme_common to website ?

Reply:
@smottet I don't think anything should be moved to theme_common, this tour is a feature of website. The related ir.asset should just be enabled... just for the current website. Actually, maybe a bit sad that this is not automatic like for asset views as before (actually the bug you are fixing here was probably not there in your original implementation when new assets were not merged yet). Maybe check with the developer in charge to know if there is a reason that this is not automatic and/or if there is an utility function to do that (duplicate the ir.asset, with a website_id).